src/main/resources/staff/templates/email/defendantResponse/body.txt

```diff
@@ -6,7 +6,7 @@ The defendant has submitted an already paid defence which is attached as a PDF.
 You need to ask the claimant if they want to proceed with the claim.
 {% endif %}
 
-{% if response.freeMediation == 'YES' %}
+{% if response.freeMediation.orElse('NO')  == 'YES' %}
```

Comment:
I am not sure this is right. For the part of the response journey where defendant is not able to choose we should not say `The defendant has chosen not to use the free mediation service.`. In this case they dod not make any choice. Better check with business.

Reply:
isn't this what I said? 👅

Reply:
It is however at a time of writing I wasn't able to see your comment 👅